src/main/cpp/chassis/SwerveChassis.h

```diff
@@ -122,6 +122,7 @@ class SwerveChassis : public IChassis, public LoggableItem
     ISwerveDriveState *GetSpecifiedDriveState(ChassisOptionEnums::DriveStateType driveOption);
 
     bool IsRotating() const { return m_isRotating; }
+    double GetRotationRateDegreesPerSecond() const { return m_pigeon != nullptr ? m_pigeon->GetRate() : 0.0; }
```

Comment:
You didn't use this in SwerveChassis.cpp

Line 239 is still this:
 auto useVision = (m_pigeon != nullptr && std::abs(m_pigeon->GetRate()) < 720.0);

Reply:
Updated here and a couple of other places to use the new call